packages/devtools_app/lib/src/charts/chart_controller.dart

```diff
@@ -104,13 +104,13 @@ class ChartController extends DisposableController
   final bool displayYLabels;
 
   /// xCanvas coord for plotting data.
-  double xCanvasChart;
+  double? xCanvasChart;
 
   /// Width of the canvas for plotting data (#).
-  double canvasChartWidth;
+  double? canvasChartWidth;
 
   /// Right-side padding after computing minPadding and max number of integral ticks for canvasChartWidth.
-  double xPaddingRight;
+  double? xPaddingRight;
```

Comment:
can these all be late?

Reply:
Nope as it is public filed that can be accessed before initialization